scratch/app-zoom.jsonc

```diff
@@ -0,0 +1,73 @@
+{
+  "type": "view",
+  "bindings": [
+    {
+      "type": "view",
+      "bindings": [
+        {
+          "type": "markdown",
+          "label": "##### Your Personal Meeting ID\n440 287 4306\nhttps://mattermost.zoom.us/my/matt.birtch"
+        },
+        {
+          "type": "button",
+          "label": "Start a Meeting",
+          "icon": "icons/zoom-icon-small.png",
+          "submit": {
+            "path": "actions/start_meeting",
+            "state": {
+              "meeting_id": "4402874306"
```

Comment:
In the Zoom design screenshot, there seems to be a button that is also a select dropdown. How important is it to support this kind of component? What is expected to be in the dropdown for this button/select in the Zoom design?

Reply:
This was to choose whether you want to use your Personal meeting ID or to create a unique meeting ID....not a necessary requirement, but we could approach this differently (using a checkbox under the button maybe). 

It was intended to be something similar to what the Zoom app has here:
<img width="418" alt="image" src="https://user-images.githubusercontent.com/2040554/186914630-780300ad-6fb9-410b-a516-6c18a2d52f8b.png">

Again, we can accomplish this in a different way probably. Most of the designs referenced here were POC designs and didn't dive extremely deep in to each app